src/PurgeClient/LocalPurgeClient.php

```diff
@@ -22,7 +23,7 @@ class LocalPurgeClient implements PurgeClientInterface
      */
     protected $cacheStore;
 
-    public function __construct(RequestAwarePurger $cacheStore)
+    public function __construct(Psr6StoreInterface $cacheStore)
```

Comment:
Can we drop LocalPurgeClient?

Reply:
Both `LocalPurgeClient` and `VarnishPurgeClient` are now kept in sake of common interface for prefix decorator.

Reply:
ok, so keep for now until we can find another way to prefix tags